### PR TITLE
Cleaned the code a little bit.

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: dashboard/load_dashboard.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 """Loads visualisations to constantly running dashboard."""
 
-import pandas as pd
```
Comment (Leon-JS02) [inline dashboard/load_dashboard.py:3]: Good job spotting this unneeded import.

File: short_term/async_api_call.py [hunk 2]
```diff
@@ -1,10 +1,14 @@
+"""This scripts sends all of the calls to the API efficiently using grequests."""
+
 import grequests
 
 
 BASE_URL = "https://data-eng-plants-api.herokuapp.com/plants/"
 
 
 def get_plant_data(plant_ids: list[int]):
+    """Sends simultaneous calls to the API."""
```
Comment (Leon-JS02) [inline short_term/async_api_call.py:10]: Nice docstring addition.

**Review by Leon-JS02 — APPROVED** [review]
Nice changes to code style/cleanliness.

File: dashboard/extract_dashboard.py [hunk 3]
```diff
@@ -30,7 +30,8 @@ def extract_readings():
         schema_name = ENV['SCHEMA_NAME']
 
         query = f'''
-        SELECT plant_id, botanist_id, at, last_watered, soil_moisture, temperature FROM {schema_name}.reading
+        SELECT plant_id, botanist_id, at, last_watered,
```
Comment (YakMan101) [inline dashboard/extract_dashboard.py:33]: Nice formatting

**Review by YakMan101 — APPROVED** [review]

**Pull request merged**